database/sqlite/sqlite_aclk.c

```diff
@@ -107,15 +205,13 @@ static int is_host_available(uuid_t *host_id)
 }
 
 // OPCODE: ACLK_DATABASE_DELETE_HOST
-void sql_delete_aclk_table_list(struct aclk_database_worker_config *wc, struct aclk_database_cmd cmd)
+static void sql_delete_aclk_table_list(char *host_guid)
 {
-    UNUSED(wc);
     char uuid_str[GUID_LEN + 1];
```

Comment:
You did not touch this code, but to improve code  I suggest to change `GUID_LEN +1` to `UUID_STR_LEN`.